airbyte-webapp/src/components/ProgressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,234 @@
+          numeratorRecords += stream.stats.recordsEmitted ?? 0;
+          denominatorRecords += stream.stats.estimatedRecords ?? 0;
+          numeratorBytes += stream.stats.bytesEmitted ?? 0;
+          denominatorBytes += stream.stats.estimatedBytes ?? 0;
+        }
+      }
+    } else if (!latestAttempt.streamStats) {
+      displayProgressBar = false;
+    }
+  }
+
+  totalPercentRecords = denominatorRecords > 0 ? Math.floor((numeratorRecords * 100) / denominatorRecords) : 0;
+
+  // chose to estimate time remaining based on records rather than bytes
+  if (latestAttempt && latestAttempt.status === Status.RUNNING && displayProgressBar) {
+    elapsedTimeMS = new Date().getTime() - latestAttempt.createdAt * 1000;
+    timeRemaining = Math.floor(elapsedTimeMS / totalPercentRecords) * (100 - totalPercentRecords); // in ms
+    const minutesRemaining = Math.ceil(timeRemaining / 1000 / 60);
+    const hoursRemaining = Math.ceil(minutesRemaining / 60);
+    if (minutesRemaining <= 60) {
+      timeRemainingString = `${minutesRemaining} ${formatMessage({ id: "estimate.minutesRemaining" })}`;
+    } else {
+      timeRemainingString = `${hoursRemaining} ${formatMessage({ id: "estimate.hoursRemaining" })}`;
+    }
+  }
+
+  return (
+    <div className={classNames(styles.container)}>
+      {displayProgressBar && <Line percent={totalPercentRecords} strokeColor={[color]} />}
+      {latestAttempt?.status === Status.RUNNING && (
+        <>
+          {displayProgressBar && (
+            <div>
+              {totalPercentRecords}% {timeRemaining < Infinity && timeRemaining > 0 ? `| ~${timeRemainingString}` : ""}
+            </div>
+          )}
+          {!displayProgressBar && unEstimatedStreams.length > 0 && (
+            <div>
+              {unEstimatedStreams.length} {formatMessage({ id: "estimate.unEstimatedStreams" })}
+            </div>
+          )}
+          {denominatorRecords > 0 && (
+            <>
+              <div>
+                {formatNumber(numeratorRecords)} {displayProgressBar ? "" : `/ ${formatNumber(denominatorRecords)}`}{" "}
+                {formatMessage({ id: "estimate.recordsSynced" })} @{" "}
+                {Math.round((numeratorRecords / elapsedTimeMS) * 1000)}{" "}
+                {formatMessage({ id: "estimate.recordsPerSecond" })}
+              </div>
+              <div>
+                {formatBytes(numeratorBytes)}{" "}
+                {displayProgressBar ? (
+                  ""
+                ) : (
+                  <>
+                    <span>/ </span>
+                    {formatBytes(denominatorBytes)}
+                  </>
+                )}{" "}
+                {formatMessage({ id: "estimate.bytesSynced" })} @ {formatBytes((numeratorBytes * 1000) / elapsedTimeMS)}
+                {formatMessage({ id: "estimate.perSecond" })}
+              </div>
+            </>
+          )}
+
+          {latestAttempt.streamStats && !showStreams && (
+            <div>
+              <br />
```

Comment:
`<br>` should not be used. This should be handled via styling and using appropriate paddings/margins.